Source/SwiftUI/UploadInfoView.swift

```diff
@@ -171,7 +173,7 @@ struct UploadInfoView: View {
 
 	private func warning(_ info: StaticString, shouldDisplay: Bool) {
 		if shouldDisplay {
-			os_log(info, type: .default)
+            logger.info("\(info)")
```

Comment:
`info` is already a string so there is no need to wrap it in a string and interpolate it again, I think, unless the `.info()` function cannot take a `StaticString` directly.

```suggestion
            logger.info(info)
```